src/murmur/Server.cpp

```diff
@@ -1373,8 +1373,9 @@ void Server::newClient() {
 		sock->setPrivateKey(qskKey);
 		sock->setLocalCertificate(qscCert);
 
-		QSslConfiguration config = sock->sslConfiguration();
+		QSslConfiguration config;
 #if QT_VERSION >= QT_VERSION_CHECK(5,15,0)
+        config = sock->sslConfiguration();
```

Comment:
Wrong indentation (spaces instead of tabs).

Reply:
my bad I saw it as soon as I created the pull request. Should be fixed now.